library/amazoncorretto

```diff
@@ -0,0 +1,5 @@
+Maintainers: Eric Edens (@ericedens)
+GitRepo: https://github.com/corretto/corretto-8-docker.git
+
+Tags: 8, 8u192, 8-al2-full, latest
+GitCommit: d654b54d9d01d7867a4e2141eb5515f2ff54cab7
```

Comment:
The Dockerfile in this commit doesn't lock the yum install to a specific version, so later rebuilds on this commit will build a newer version than implied here (8u192).

Reply:
Good call. I added a branch for the 8-al2-full series, and then added a commit (https://github.com/corretto/corretto-8-docker/commit/e83e992e4f6227de1562c3665cfd32d4639fba9a) specifically for 8u192 within that series.